users/api/requests.go

```diff
@@ -39,15 +38,16 @@ func (req createUserReq) validate() error {
 		return err
 	}
 	// Username must not be a valid email format due to username/email login.
-	if _, err := mail.ParseAddress(req.User.Credentials.Username); err == nil {
+	if err := api.ValidateEmail(req.User.Email); err == nil {
 		return apiutil.ErrInvalidUsername
 	}
+
```

Comment:
```suggestion
	if err := api.ValidateEmail(req.User.Credentials.Username); err == nil {
		return apiutil.ErrInvalidUsername
	}

```

this logic is to ensure that a user does not have an email address as a username